### PR TITLE
Cleanup CI

### PR DESCRIPTION
**Pull request by vatanaksoytezer**
Cleans up the GH action configuration files

### CODE REVIEW

File: moveit2_tutorials.repos [hunk 1]
```diff
@@ -1,9 +1,5 @@
 repositories:
-  moveit2_tutorials:
```
Comment (MarqRazz) [inline moveit2_tutorials.repos:2]: Do any of the instructions need to be updated by removing this? I think the readme expects the repo itself to be included in this file. I will try to dig around a little and see.

**Review by MarqRazz — COMMENTED** [review]
What did we replace `htmlcheck.yaml` with?

File: moveit2_tutorials.repos (hunk 1 above)
Comment (vatanaksoytezer) [reply moveit2_tutorials.repos:2]: Good catch, fixing it.

File: README.md [hunk 2]
```diff
@@ -42,6 +42,7 @@ Download Moveit2_tutorials Source Code
 
     wget https://raw.githubusercontent.com/ros-planning/moveit2_tutorials/main/moveit2_tutorials.repos
     vcs import < moveit2_tutorials.repos
+    git clone https://github.com/ros-planning/moveit2_tutorials.git
     rosdep install -r --from-paths . --ignore-src --rosdistro foxy -y
```
Comment (MarqRazz) [inline README.md:46]: Sorry I didnt say but we should change this to clone `moveit2_tutorials` and then extract the `.repos` from that folder and not use `wget`.

```
git clone https://github.com/ros-planning/moveit2_tutorials.git
vcs import < moveit2_tutorials/moveit2_tutorials.repos
rosdep install -r --from-paths . --ignore-src --rosdistro foxy -y
```

Comment (vatanaksoytezer) [reply README.md:46]: Sure

**Comment by MarqRazz** [comment]
Vatan, any idea on why the `CheckBuildHTML` action is not reporting success?
image

**Comment by vatanaksoytezer** [comment]
> Vatan, any idea on why the `CheckBuildHTML` action is not reporting success?
> image

I changed the name of it to Build Pages

**Comment by vatanaksoytezer** [comment]
> What did we replace `htmlcheck.yaml` with?

Initially I thought of removing it with parallel to what we did in moveit_tutorials with Robert, but I think keeping it does not harm and reasonable.

**Comment by MarqRazz** [comment]
> > Vatan, any idea on why the `CheckBuildHTML` action is not reporting success?
> > image
> 
> I changed the name of it to Build Pages

Do you know why `CheckBuildHTML` is still in the Action tests list and showing the yellow running state? Is it because that test ran when you started the PR so now it always expects it?

If you changed the name do we need to update the badges on the README?

**Comment by vatanaksoytezer** [comment]
> Is it because that test ran when you started the PR so now it always expects it?

That is the reason. Once this PR is merged the action will be replaced with the new action and will no longer be expected. We would need to remove / delete the previous runs in the action list in Github for full cleanup though. I will manually delete that action from the list after the merge. The name of the file has not been changed, so no need for badge updates. The internal name of the action is the that got changed

**Review by MarqRazz — APPROVED** [review]
It looks like we dont have a badge for the HTML check if you feel like adding one.

Thanks for getting this done it looks much cleaner.

**Comment by vatanaksoytezer** [comment]
It should be much cleaner with the last commit now. We need someone with admin access to change the required actions to new ones to merge this.

File: .github/workflows/industrial_ci_action.yml [hunk 3]
```diff
@@ -11,85 +11,38 @@ on:
...
-          df -h
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
       - name: cache upstream_ws
         uses: pat-s/always-upload-cache@v2.1.3
         with:
           path: ${{ env.BASEDIR }}/upstream_ws
-          key: upstream_ws-${{ env.CACHE_PREFIX }}-${{ hashFiles('moveit2.repos') }}-${{ github.run_id }}
+          key: upstream_ws-${{ env.CACHE_PREFIX }}-${{ hashFiles('moveit2_tutorials.repos.repos') }}-${{ github.run_id }}
```
Comment (rhaschke) [inline .github/workflows/industrial_ci_action.yml:40]: This seems to be the wrong file name: *.repos.repos
```suggestion
          key: upstream_ws-${{ env.CACHE_PREFIX }}-${{ hashFiles('moveit2_tutorials.repos') }}-${{ github.run_id }}
```

File: .github/workflows/industrial_ci_action.yml [hunk 4]
```diff
@@ -11,85 +11,38 @@ on:
...
           restore-keys: |
-            upstream_ws-${{ env.CACHE_PREFIX }}-${{ hashFiles('moveit2.repos') }}
+            upstream_ws-${{ env.CACHE_PREFIX }}-${{ hashFiles('moveit2_tutorials.repos.repos') }}
```
Comment (rhaschke) [inline .github/workflows/industrial_ci_action.yml:42]: Same here.
```suggestion
            upstream_ws-${{ env.CACHE_PREFIX }}-${{ hashFiles('moveit2_tutorials.repos') }}
```

**Review by rhaschke — APPROVED** [review]
Some small issue(s) to fix remain. Otherwise, I approve.

**Comment by rhaschke** [comment]
@henningkayser, I guess you have admin rights here to remove CheckBuildHTML as a required CI job.

**Comment by MarqRazz** [comment]
So I removed `CheckBuildHTML` from the list of "Status checks that are required" under the branch protection rules but still cant get the option to merge it.
image

I can try to disable `Require status checks to pass before merging` and re-enable it after if you guys think that is a good option.

**Comment by vatanaksoytezer** [comment]
> I can try to disable `Require status checks to pass before merging` and re-enable it after if you guys think that is a good option.

That looks reasonable we can enable it right back

**Pull request merged**